source/CardGames.py

```diff
@@ -144,6 +144,9 @@ def add_guess(self, guess):
     else:
       self.guesses.append(guess)
 
+  def show_guesses(self):
```

Comment:
Looks good brother. Just stylistically, can you add a comment specifying that this is the Marshall's past guesses, not her hand. Thanks!

Reply:
Ah yes good suggestion brother, will do.